### DISCUSSION TITLE
Using ### {{weekEvents}} ### makes model to hallucinate

### DISCUSSION THREAD

**Discussion by danceballos** [SKaplanOfficial/Raycast-PromptLab#19] [General]
I am trying to write a PL Commands that checks for available times to have a meeting, here is the prompt: 

```
Take into account the events list below as non available times. Given today is {{date}} list available times for meetings for the next 5 working days. 

Also, consider I work form 9 AM and 6 PM

Please provide the available meeting slots, considering the provided information.

Desired format (example): 
{date} from {starting hour} to {ending hour}

Events list: ###
{{weekEvents}}
###
```

The output keeps having time slots on top of meeting times and some times it completely ignores open times. 

Anyone has a simmilar issue?

**Comment by SKaplanOfficial** [comment]
That could be a few different things. To rule out problems, can you share an example of the fully substituted prompt (command+shift+p on the response view, & remove any personal info). Just want to make sure it's getting the week events properly.

It also depends on which model you're using.
